WalletWasabi/Backend/Models/FilterModel.cs

```diff
@@ -31,6 +30,12 @@ public FilterModel(SmartHeader header, Lazy<GolombRiceFilter> filter)
 	// is constructed.This ensures the key is deterministic while still varying from block to block.
 	public byte[] FilterKey => Header.BlockHash.ToBytes()[..16];
 
+	public static FilterModel FromParameters(uint blockHeight, uint256 blockHash, byte[] filterData, uint256 prevBlockHash, long blockTime)
```

Comment:
I would use the name `Create` because it is the most common one. Anyway, it is not important but it would make it more natural.